missing_numbers.py

```diff
@@ -0,0 +1,14 @@
+def missing_number(b):
+    if len(b) != 0:
+        first_list = []
+        for x in range(b[0], b[-1]+1):
+            first_list.append(x)
+        new_b = set(b)
+        
+        return (list(new_b ^ set(first_list)))
+    else:
```

Comment:
This else statement is redundant. i.e When a function meets a `return` statement, it stops executing i.e
```py
# instead of writing
if some_serious_condition:
  return some_cute_value
else:
  return some_annoying_value

# you can write
if some_serious_condition:
  return some_cute_value
# this should be out of the if block 
return some_annoying_value
```

Reply:
This is awesome, have just done it. Thanks @dennisja, I didn't know about it